my-code/movie-search/src/index.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { Provider as Store } from "react-redux";
```

Comment:
Save the redux things in a different file, then you will split the responsibilities of this file